docs/Documentation/UserGuideV0.7.0/2-Concept Key Concepts and Terminology/1-Key Concepts and Terminology.md

```diff
@@ -95,12 +95,12 @@ When `*` appears in the middle of the path, it represents `*` itself, i.e., a la
 
 * Timestamp
 
-The timestamp is the time point at which a data arrives. IoTDB timestamps are divided into two types: LONG and DATETIME (including DATETIME-INPUT and DATETIME-DISPLAY). When a user enters a timestamp, he can use a LONG type timestamp or a DATETIME-INPUT type timestamp, where the support format of the DATETIME-INPUT type timestamp is shown in Table 2-1.
+The timestamp is the time point at which a data arrives. IoTDB timestamps are divided into two types: LONG and DATETIME (including DATETIME-INPUT and DATETIME-DISPLAY). When a user enters a timestamp, he can use a LONG type timestamp or a DATETIME-INPUT type timestamp, where the support format of the DATETIME-INPUT type timestamp is shown in table:
```

Comment:
I suggest "at which data is produced" over "at which a data arrives" because data arrival time for a database is just another thing.
"user enters" -> "user inputs"
"where the support format of the DATETIME-INPUT type timestamp is shown in table" -> "and the supported formats of the DATETIME-INPUT type timestamp are shown in the table below"